Tasks/IISWebAppManagementOnMachineGroup/task.loc.json

```diff
@@ -76,6 +76,14 @@
     }
   ],
   "inputs": [
+    {
+      "name": "DeployIIS",
```

Comment:
Is this EnableIIS or DeployIIS? with Yaml exposing inputs, the names also have to be meaningful. Can you sync-up with @RoopeshNair to ensure this new input name is fine.